arch/arm/src/arm/Toolchain.defs

```diff
@@ -51,20 +51,22 @@ endif
 #  CROSSDEV         The GNU toolchain triple (command prefix)
 #  ARCHCPUFLAGS     CPU-specific flags selecting the instruction set
 #                   FPU options, etc.
-#  MAXOPTIMIZATION  The maximum optimization level that results in
+#  ARCHOPTIMIZATION The maximum optimization level that results in
```

Comment:
is it still the `The maximum optimization level`?

Reply:
Remove "maximum"